srt-tokio/src/socket.rs

```diff
@@ -201,6 +222,14 @@ impl SrtSocket {
     pub fn settings(&self) -> &ConnectionSettings {
         &self.settings
     }
+
+    pub fn stats(&mut self) -> &mut impl Stream<Item = SocketStatistics> {
```

Comment:
The reference implementation also has a way to clear statistics--which I believe is the default when you get the statistics. We should have some sort of mechanism for that

Reply:
This can be handled as part of a separate issue, specifically one dealing with a C API. My main goal was to just get something working so we could iterate on the quality of the metrics while we figure out the ergonomics of the API.